core/mutator/entry/mutator.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
If this is branched out of `core/mutator/entry/entry.go`, can you please use `git mv`? It might make it easier to review.